xrally_kubernetes/platforms/existing.py

```diff
@@ -46,14 +97,10 @@ def cleanup(self, task_uuid=None):
             "errors": []
         }
 
-    def check_health(self):
-        """Check whatever platform is alive."""
-        # TBD
-        return {"available": True}
-
-    def info(self):
-        """Return information about cloud as dict."""
-        return {"info": {}}
-
     def _get_validation_context(self):
         return {}
+
+    def info(self):
+        service = kubernetesclient.K8sClient(self.platform_data)
+        version = version_api.VersionApi(service.api).get_code().to_dict()
```

Comment:
it would be nice to move it under service class, so all logic for getting and processing k8s data will be concentrated in one place.